### PR TITLE
Angular app portal documentation update

### PR DESCRIPTION
**Pull request by Oluwadaminiola**
updated angular app-portal documentation, took out vue app portal documentation

### CODE REVIEW

File: web/ui/website/content/docs/app-portal.md [hunk 1]
```diff
@@ -24,18 +24,55 @@ App portal to be usable in three different ways:
 As explained above, the iframe code snippet was made available as the easiest to present app portal to your customers. The token embedded into the iframe code expires, so you can use this [API](https://convoy.readme.io/reference/post_security-applications-appid-keys) to generate a new token whenever your user enters the page with the iframe. Simply replace the `key` gotten from the API response with `{token}` in the example below.
 
 ```html[iframe snippet]
-<iframe style="width: 100%; height: 100vh; border: none;" src="http://localhost:4200/ui/app-portal/{ token }&appId=291e98cb-4e93-408f-bb5b-d422ff13d12c"></iframe>
+<iframe style="width: 100%; height: 100vh; border: none;" src="{ apiURL }/app-portal/{ token }&appId=291e98cb-4e93-408f-bb5b-d422ff13d12c"></iframe>
```
Comment (horlah) [inline web/ui/website/content/docs/app-portal.md:27]: Please update the ending of the URL here to `&appId={ app id }` and add the explanation of what `appId` and `apiURL` are in the description above.

**Review by horlah — CHANGES_REQUESTED** [review]

File: web/ui/website/content/docs/app-portal.md (hunk 1 above)
Comment (Oluwadaminiola) [reply web/ui/website/content/docs/app-portal.md:27]: Noted

Comment (horlah) [reply web/ui/website/content/docs/app-portal.md]: You missed the description of the `apiURL` @Oluwadaminiola

**Comment by horlah** [comment]
@Oluwadaminiola The sidebar of the documentation is not loading the subpages. Is it loading on your machine?

**Review by horlah — APPROVED** [review]

**Pull request merged**